src/Validator/Range.php

```diff
@@ -92,6 +108,11 @@ public function isArray(): bool
      */
     public function getType(): string
     {
+        if($this->format === 'int32') {
+            return self::TYPE_INTEGER;
+        } else if($this->format === 'float') {
+            return self::TYPE_FLOAT;
+        }
```

Comment:
```suggestion
        if($this->format === 'float') {
            return self::TYPE_FLOAT;
        }
```

If we return integer anyway, no need to check at first.